x-pack/legacy/plugins/uptime/public/lib/alert_types/monitor_status.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ThrowReporter } from 'io-ts/lib/ThrowReporter';
+import React from 'react';
+import DateMath from '@elastic/datemath';
+import { isRight } from 'fp-ts/lib/Either';
+import {
+  AlertTypeModel,
+  ValidationResult,
+  // TODO: this typing issue should be resolved after NP migration
+  // eslint-disable-next-line @kbn/eslint/no-restricted-paths
+} from '../../../../../../plugins/triggers_actions_ui/public/types';
+import { AlertTypeInitializer } from '.';
+import { StatusCheckExecutorParamsType } from '../../../common/runtime_types';
+import { AlertMonitorStatus } from '../../components/connected/alerts';
+
+/**
+ * Check if params have properties.
+ * @param params the alert params
+ */
+const hasProps = (params: any): boolean => {
+  for (const p in params) {
+    if (Object.prototype.hasOwnProperty.call(params, p)) {
+      return true;
+    }
+  }
+  return false;
+};
```

Comment:
could have just simple user Object.key().length or loadash.isEmpty

Reply:
Yeah, I debated this myself when I was writing the code. My reasoning was that this might be a faster solution, but I think it's not worth the added code size. I will change it to use `Object.keys`.

Reply:
Resolved: https://github.com/elastic/kibana/pull/57919/commits/6584d3fbdfb0d4b93ea16ce0c60184ae190e1e2d